src/libslic3r/PrintConfig.cpp

```diff
@@ -4920,10 +4920,10 @@ void PrintConfigDef::init_fff_params()
     def->mode     = comAdvanced;
     def->set_default_value(new ConfigOptionBool(false));
 
-    def = this->add("support_remove_small_overhang", coBool);
-    def->label = L("Remove small overhangs");
+    def = this->add("support_ignore_small_overhang", coBool);
```

Comment:
Let's avoid renaming the variable. The extensive changes required are not justified. While `ignore` is clearer, `support_remove_small_overhang` is not entirely incorrect, as it represents "remove support spots that are small overhangs." It may not be straightforward for end users (and shouldn't be used by them).